src/pages/Transfer/TransferForm/index.tsx

```diff
@@ -0,0 +1,220 @@
+      [TRANSFER_AMOUNT]: '',
+      [RECIPIENT_ADDRESS]: ''
+    });
+  }, [
+    submitStatus,
+    reset,
+    t
+  ]);
+
+  return (
+    <>
+      <form
+        className='space-y-8'
+        onSubmit={handleSubmit(onSubmit)}>
+        <FormTitle>
+          {t('transfer_page.transfer_currency')}
+        </FormTitle>
+        <div>
+          <p
+            className={clsx(
+              'mb-2',
+              'text-right',
+              { 'text-interlayDenim':
+        process.env.REACT_APP_RELAY_CHAIN_NAME === POLKADOT },
+              { 'dark:text-kintsugiOchre':
+        process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA }
+            )}>Balance: {activeToken?.balance}
+          </p>
+          <div
+            className={clsx(
+              'flex',
+              'gap-2'
+            )}>
+            {/* TODO: use forwardRef to pull in select value as form data */}
+            <Tokens
+              variant='formField'
+              showBalances={false}
+              callbackFunction={handleTokenChange} />
+            <TokenAmountField
+              id={TRANSFER_AMOUNT}
+              name={TRANSFER_AMOUNT}
+              ref={register({
+                required: {
+                  value: true,
+                  message: t('redeem_page.please_enter_amount')
+                },
+                validate: value => validateTransferAmount(value)
+              })}
+              error={!!errors[TRANSFER_AMOUNT]}
+              helperText={errors[TRANSFER_AMOUNT]?.message} />
+          </div>
+        </div>
+        <div>
+          <TextField
+            id={RECIPIENT_ADDRESS}
+            name={RECIPIENT_ADDRESS}
+            type='text'
+            label={t('recipient')}
+            placeholder={t('recipient_account')}
+            ref={register({
+              required: {
+                value: true,
+                message: t('enter_recipient_address')
+              },
+              validate: value => validateAddress(value)
+            })}
+            error={!!errors[RECIPIENT_ADDRESS]}
+            helperText={errors[RECIPIENT_ADDRESS]?.message} />
+        </div>
+        <SubmitButton
+          disabled={
+            parachainStatus !== ParachainStatus.Running
+          }
+          pending={submitStatus === STATUSES.PENDING}
+          onClick={handleConfirmClick}>
+          {accountSet ? (
+            t('transfer')
+          ) : (
+            t('connect_wallet')
+          )}
+        </SubmitButton>
+      </form>
+      {(submitStatus === STATUSES.REJECTED && submitError) && (
+        <ErrorModal
+          open={!!submitError}
+          onClose={() => {
+            setSubmitStatus(STATUSES.IDLE);
+            setSubmitError(null);
+          }}
+          title='Error'
+          description={
+            typeof submitError === 'string' ?
+              submitError :
+              submitError.message
+          } />
+      )}
+    </>
+  );
+};
+
+export default TransferForm;
```

Comment:
@tomjeatt 
It looks like this component is not wrapped in a react error boundary.

Reply:
Nope :) let me fix that